snippets/worker-latency-profile-intro.adoc

```diff
@@ -8,16 +8,20 @@
 
 :_content-type: SNIPPET
 
-All nodes send heartbeats to the Kubernetes Controller Manager Operator (kube controller) in the {product-title} cluster every 10 seconds, by default. If the cluster does not receive heartbeats from a node, {product-title} responds using several default mechanisms.
+`Kublet` sets status for all nodes which is read by the Kubernetes Controller Manager (kube controller) in the {product-title} cluster every 10 seconds, by default. If the kube controller does not receive status from a node, {product-title} responds using several default mechanisms.
 
-For example, if the Kubernetes Controller Manager Operator loses contact with a node after a configured period:
+For example, if the Kubernetes Controller Manager loses contact with a node after a configured period, the default behavior is:
 
 . The node controller on the control plane updates the node health to `Unhealthy` and marks the node `Ready` condition as `Unknown`.
 
 . In response, the scheduler stops scheduling pods to that node.
 
-. The on-premise node controller adds a `node.kubernetes.io/unreachable` taint with a `NoExecute` effect to the node and schedules any pods on the node for eviction after five minutes, by default.
+. The Node Lifecycle Controller adds a `node.kubernetes.io/unreachable` taint with a `NoExecute` effect to the node and schedules any pods on the node for eviction after five minutes, by default.
 
-This behavior can cause problems if your network is prone to latency issues, especially if you have nodes at the network edge. In some cases, the Kubernetes Controller Manager Operator might not receive an update from a healthy node due to network latency. The Kubernetes Controller Manager Operator would then evict pods from the node even though the node is healthy. To avoid this problem, you can use _worker latency profiles_ to adjust the frequency that the kubelet and the Kubernetes Controller Manager Operator wait for status updates before taking action. These adjustments help to ensure that your cluster runs properly in the event that network latency between the control plane and the worker nodes is not optimal.
+This behavior can cause problems if your network is prone to latency issues, especially if you have nodes at the network edge. In some cases, the Kubernetes Controller Manager might not receive an update from a healthy node due to network latency. The Kubernetes Controller Manager would then evict pods from the node even though the node is healthy. 
```

Comment:
Also, the actual eviction is not carried out by the KCM. You can drop the reference that indicates KCM is responsible for it.  

For example, check how we have worded it in enhancement - https://github.com/openshift/enhancements/blob/master/enhancements/worker-latency-profile/worker-latency-profile.md#default-update-and-default-reaction

Reply:
@harche Thank you! Glad to do so!

Reply:
The actual eviction process is handled by the kubelet.

Reply:
@harche I've made this change in the "introduction text". It no longer claims KCM is responsible for eviction.